VERSIONING.md

```diff
@@ -31,8 +39,9 @@ Any non-backward compatible changes leads to a major version bump. This includes
 - Any structural change in the CLI output
 - Any object, property or function that is removed from the serverless object passed to plugins
 - Remove an event from the list of lifecycle events of core commands
+- Drop of supoprt for some major Node.js  version
```

Comment:
> Drop of support for some major Node.js versions

Maybe add a note / link to [Node.js Release schedule](https://github.com/nodejs/Release/blob/master/README.md)?